pkg/trace/event/processor.go

```diff
@@ -50,20 +50,13 @@ func (p *Processor) Stop() {
 
 // Process takes a processed trace, extracts events from it and samples them, returning a collection of
 // sampled events along with the total count of extracted events.
-func (p *Processor) Process(root *pb.Span, t pb.Trace) (events []*pb.Span, numExtracted int64) {
-	if len(p.extractors) == 0 {
-		return
-	}
-
-	priority, hasPriority := sampler.GetSamplingPriority(root)
-	if !hasPriority {
-		priority = sampler.PriorityNone
-	}
-
+func (p *Processor) Process(root *pb.Span, t *pb.TraceChunk, keptTrace bool) (numEvents, numExtracted int64) {
```

Comment:
Can you update the doc to explain what `keptTrace` is and why it's there please? Also, since `t` has a field called `DroppedTrace`, is it still needed?

Reply:
You are right. We don't need this argument anymore. We can just use `t.DroppedTrace`.